server/graphql/general/src/queries/store.rs

```diff
@@ -138,6 +138,7 @@ impl StoreFilterInput {
         StoreFilter {
             id: id.map(EqualFilter::from),
             code: code.map(StringFilter::from),
+            name_id: None,
```

Comment:
Nit: Tending to exclude this `id` filter if not need...

Reply:
Sorry how do you exclude? Compiler was annoyed with me for not specifying it.

Reply:
I could do `..Default::default()` but seems verbose? And if adding more filters in the future it might make us miss it in gql without the compiler complaining?

Reply:
sorry I read it wrong all good